### DISCUSSION TITLE
How to specify the order of screens in a tab layout?

### DISCUSSION THREAD

**Discussion by rememberlenny** [expo/router#47] [Q&A] [answered]
How do you set the tab order in the demo app example?

**Comment by trin4ik** [comment, marked as answer]
> How do you set the tab order in the demo app example?

check file [`app/(root)/(tab).js`](https://github.com/expo/router/blob/main/apps/demo/app/(root)/(tab).js)
u can only export `Tabs`, like this:

```javascript
import { Tabs } from "expo-router";
export default Tabs;
```

but this way automatically generated tabbars. if u need your own order, u need listed it in `<Tabs>` component, like:

```javascript
import { Tabs } from "expo-router";
export default function Layout1() {
  return (
    <Tabs>
      <Tabs.Screen name="settings" />
      <Tabs.Screen name="index" />
    </Tabs>
  );
}
```

`settings` first, `index` last. but u not need listed all other tabs, because expo-router listed it automatically. finnaly, all u need to set specific tab first is just:

```javascript
import { Tabs } from "expo-router";
export default function Layout1() {
  return (
    <Tabs>
      <Tabs.Screen name="settings" />
    </Tabs>
  );
}
```

**Comment by EvanBacon** [comment]
This behavior is currently outlined in the docs https://expo.github.io/router/docs/features/options open to PRs adding algolia search to the docs.

**Reply by mwmcode** [reply]
Not anymore, do you happen to know where the docs live today?

**Comment by rememberlenny** [comment]
Thank you
